benefits/static/css/styles.css

```diff
@@ -705,22 +708,12 @@ footer.global-footer .footer-links a:active {
   }
 
   .media-list .media .media-body--heading {
-    font-size: 20px;
     line-height: 30px;
```

Comment:
Should `line-height` be a variable?

Reply:
This PR is only dealing with paragraphs and links, so it will in a future PR.

Reply:
I would argue `line-height` is part of the font definition, but leave it up to you.